prowler/providers/azure/services/defender/defender_auto_provisioning_log_analytics_agent_vms_on/defender_auto_provisioning_log_analytics_agent_vms_on.py

```diff
@@ -0,0 +1,24 @@
+from prowler.lib.check.models import Check, Check_Report_Azure
+from prowler.providers.azure.services.defender.defender_client import defender_client
+
+
+class defender_auto_provisioning_log_analytics_agent_vms_on(Check):
+    def execute(self) -> Check_Report_Azure:
+        findings = []
+
+        for subscription, auto_provisioning_settings in defender_client.auto_provisioning_settings.items():
+            report = Check_Report_Azure(self.metadata())
+            report.status = "PASS"
+            report.subscription = subscription
+            report.resource_name = "Defender Auto Provisioning Log Analytics Agents On"
+            report.resource_id = auto_provisioning_settings["default"].resource_id
```

Comment:
Why the `default`? Just curious.

Reply:
Only checks for default workspace for this check